doc/manuals/admin/yum.md

```diff
@@ -0,0 +1,49 @@
+## Using YUM repositories
+
+There are 2 available packages:
+- nightly 
+- release
+ 
+You can manually add config for yum repo or download it from FIWARE public repository.
+
+Use this config for nightly packages:
+```
+[fiware-nightly]
+name=FIWARE nightly repository
+baseurl=http://nexus.lab.fiware.org/repository/el/$releasever/$basearch/nightly
+enabled=1
+gpgcheck=0
+priority=1
+
+```
+or download it from [FIWARE public repository](http://nexus.lab.fiware.org/repository/raw/public/repositories/el/7/x86_64/fiware-nightly.repo)
+```
+curl -o /etc/yum.repos.d/fiware-nightly.repo http://nexus.lab.fiware.org/repository/raw/public/repositories/el/7/x86_64/fiware-nightly.repo
+
+```
+
+Use this config for release packages:
+```
+[fiware-release]
+name=FIWARE release repository
+baseurl=http://nexus.lab.fiware.org/repository/el/$releasever/$basearch/release
+enabled=1
+gpgcheck=0
+priority=1
+
+```
+or download it from [FIWARE public repository](http://nexus.lab.fiware.org/repository/raw/public/repositories/el/7/x86_64/fiware-release.repo)
+```
+curl -o /etc/yum.repos.d/fiware-release.repo http://nexus.lab.fiware.org/repository/raw/public/repositories/el/7/x86_64/fiware-release.repo
+```
+
+Keep in mind, configs from FIWARE public repository have https connection and this can cause problems, to avoid them see [SSL configuration](#SSL)
+
```

Comment:
Documetnation should state if both .repo (nighly and release) can be used at the same time and, in that case, what would happend in that case (I guess that most of the times nightly will have a newer version, excep the day in which a new release is generated and uploaded).

Reply:
Ok, I will add a warning for the case of using both repos

Reply:
fixed in c6e45d153045d7d51e08a4dc3c732985c22fe5c9